websocket/src/socket.cpp

```diff
@@ -42,15 +42,24 @@ dmSocket::Result Send(WebsocketConnection* conn, const char* buffer, int length,
     }
     if (out_sent_bytes)
         *out_sent_bytes = total_sent_bytes;
+
+    DebugPrint(2, "Sent buffer:", buffer, length);
```

Comment:
For future debugging, we now have `websocket.debug = ...` that will enable more debug info.